pkg/agent/controller/ipseccertificate/ipsec_certificate_controller_test.go

```diff
@@ -290,6 +294,7 @@ func TestController_RotateCertificates(t *testing.T) {
 				assert.True(t, ok)
 				// issue a certificate with lifetime of 10 seconds.
 				signCSR(t, fakeController, csr, time.Second*10)
+				signCh <- struct{}{}
```

Comment:
Not related to this PR: should the line 288 `defer close(signCh)` right after the line 286?

Reply:
No. We need to use this channel to check whether the anonymous function in line 287 finishes as expected (return after signing two CSRs).